internal/commands/result.go

```diff
@@ -53,10 +53,15 @@ const (
 	highCx                    = "HIGH"
 	codeBashingKey            = "cb-url"
 	failedGettingBfl          = "Failed getting BFL"
-	notAvailableString        = "N/A"
+	notAvailableString        = "-"
+	scanFailedString          = "Failed"
+	scanCanceledString        = "Canceled"
+	scanSuccessString         = "Completed"
 	notAvailableNumber        = -1
+	scanFailedNumber          = -2
+	scanCanceledNumber        = -3
 	defaultPaddingSize        = -13
-	defaultResultsPaddingSize = -15
+	boldFormat                = "\033[1m%s\033[0m"
```

Comment:
what is this change?

Reply:
A cosmetic change: it's format that when using Sprintf(boldFormat,<text>) the <text> font will be **bolded**